tests/fixtures/generate.php

```diff
@@ -132,7 +147,7 @@ function _file_info($filename) {
       'user_id' => [],
     ],
     '_attachments' => [
-      'files/0/effb530e-5529-46ed-bec4-1b8f05c274d6/public/file1.txt' => _file_info(FILE1),
+      'files:0:6f9e1f07-e713-4840-bf95-8326c8317800:public:file1.txt' => _file_info(FILE1),
```

Comment:
What's the reason for changing to `:` as a delimiter? Is this so that the Drupal file path doesn't get mixed up?
How is the directory structure created in CouchDB? Would CouchDB create one single directory called e.g. `files:0:6f9e1f07-e713-4840-bf95-8326c8317800:public` and then sub-directories under that mapping to the file name e.g. `path/file1.txt` ?